java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1034,9 +1034,9 @@ public void pfadd(BaseClient client) {
     @ParameterizedTest
     @MethodSource("getClients")
     public void pfcount(BaseClient client) {
-        String key1 = UUID.randomUUID().toString();
-        String key2 = UUID.randomUUID().toString();
-        String key3 = UUID.randomUUID().toString();
+        String key1 = "{test}-hll1-" + UUID.randomUUID();
+        String key2 = "{test}-hll2-" + UUID.randomUUID();
+        String key3 = "{test}-hll3-" + UUID.randomUUID();
```

Comment:
Do we want to change these. Isn't uniform with all other tests we have.

Reply:
The test keys should have the same prefix to avoid errors.